### DISCUSSION TITLE
Porting Back To Sponge

### DISCUSSION THREAD

**Discussion by whimxiqal** [AddstarMC/Prism-Bukkit#267] [General]
Hi there,

Has there been any discussion to create a Sponge API v.8 Prism implementation based on the work done on Prism-Bukkit? We have our own fork of the original Prism on our server running Sponge API v.7 on 1.12.2, but rather than updating to v.8 ourselves for versions 1.16 and above, we would rather work towards using the many features already added to this Bukkit version.

Myself and a few other developers from our server [Minecraft Online](https://www.minecraftonline.com) would be able to help.

**Comment by Narimm** [comment]
As we break the Prism-Bukkit Version into an API I think the likely hood of this being acheivable improves...but personally I dont have the time at the moment
